src/index.ts

```diff
@@ -690,6 +717,180 @@ class Spanner extends GrpcService {
+   *     replication properties.
+   * @property {string} [baseConfig] Base configuration name,
+   *     e.g. projects/<project_name>/instanceConfigs/nam3 based on which this
+   *     configuration is created.
+   * @property {Object.<string, string>} [labels] Cloud Labels are a flexible
+   *     and lightweight mechanism for organizing cloud resources into groups
+   *     that reflect a customer's organizational needs and deployment
+   *     strategies. Cloud Labels can be used to filter collections of
+   *     resources. They can be used to control how resource metrics are
+   *     aggregated. And they can be used as arguments to policy management
+   *     rules (e.g. route, firewall, load balancing, etc.).
+   * @property {string} [etag] etag is used for optimistic concurrency control
+   *     as a way to help prevent simultaneous updates of a instance config from
+   *     overwriting each other.
+   * @property {boolean} [validateOnly] An option to validate, but not actually
+   *     execute, a request, and provide the same response.
+   */
+  /**
+   * @typedef {array} CreateInstanceConfigResponse
+   * @property {InstanceConfig} 0 The new {@link InstanceConfig}.
+   * @property {google.longrunning.Operation} 1 An operation object that can be
+   *     used to check the status of the request.
+   * @property {google.longrunning.IOperation} 2 The full API response.
+   */
+  /**
+   * @callback CreateInstanceConfigCallback
+   * @param {?Error} err Request error, if any.
+   * @param {InstanceConfig} instanceConfig The new {@link InstanceConfig}.
+   * @param {google.longrunning.Operation} operation An operation object that
+   *     can be used to check the status of the request.
+   * @param {google.longrunning.IOperation} apiResponse The full API response.
+   */
+  /**
+   * Create an instance config.
+   *
+   * Wrapper around {@link v1.InstanceAdminClient#createInstanceConfig}.
+   *
+   * @see {@link v1.InstanceAdminClient#createInstanceConfig}
+   * @see [CreateInstanceConfig API Documentation](https://cloud.google.com/spanner/docs/reference/rpc/google.spanner.admin.instance.v1#google.spanner.admin.instance.v1.InstanceAdmin.CreateInstanceConfig)
+   *
+   * @throws {GoogleError} If a name is not provided.
+   * @throws {GoogleError} If a configuration object is not provided.
+   * @throws {GoogleError} If a base config is not provided in the configuration
+   *                       object.
+   *
+   * @param {string} name The name of the instance config to be created.
+   * @param {CreateInstanceConfigRequest} config Configuration object.
+   * @param {CreateInstanceConfigCallback} [callback] Callback function.
+   * @returns {Promise<CreateInstanceConfigResponse>}
+   *
+   * @example
+   * ```
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   *
+   * const [baseInstanceConfig] =
+   *     await spanner.getInstanceConfig(baseInstanceConfigId);
```

Comment:
I think that this only gets a reference to the instance config. It does not load the metadata from the backend, which means that the `baseInstanceConfig.replicas` collection will be empty or undefined.

Reply:
getInstanceConfig is a wrapper over InstanceAdminClient#getInstanceConfig, and makes a call to the backend, which returns the metadata. I ran the sample and verified that `baseInstanceConfig.replicas` collection is non-empty.